src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -1245,4 +1249,260 @@ public SBApiResponse workflowBulkUpdateTransitionV1(String userAuthToken, Multip
+						Map<String, Object> updateKeyValue = (Map<String, Object>) valueToUpdate.get(Constants.TO_VALUE);
+						String keyToUpdate = updateKeyValue.keySet().stream().findFirst().get();
+						if(Constants.DESIGNATION.equalsIgnoreCase(keyToUpdate) || Constants.GROUP.equalsIgnoreCase(keyToUpdate)){
+							if(allPendingRequest.containsKey(wfStatusEntity.getUserId())) {
+								pendingRequestList = (List<Map<String, Object>>) allPendingRequest.get(wfStatusEntity.getUserId());
+								pendingRequestList.add(updateKeyValue);
+								allPendingRequest.put(wfStatusEntity.getUserId(), pendingRequestList);
+							} else {
+								pendingRequestList = new ArrayList<>();
+								pendingRequestList.add(updateKeyValue);
+								allPendingRequest.put(wfStatusEntity.getUserId(), pendingRequestList);
+							}
+						}
+					}
+				}
+			}
+			Map<String, Object> allUserDetails = this.getUserSearchDetails(userIdSet,false, rootOrgId);
+			if(MapUtils.isNotEmpty(allUserDetails)){
+				this.populateSheetWithPendingRequests(allPendingRequest, allUserDetails ,pendingRequestWorkBook);
+			} else {
+				return ResponseEntity.status(HttpStatus.OK).body("There are no requests pending for Group or Designation approval with the MDO");
+			}
+			String csvFilePath = "/tmp/pendingRequest.csv";
+			this.convertXSSFWorkbookToCSV(pendingRequestWorkBook, csvFilePath);
+			responseEntity = this.preparePendingRequestFileResponse(csvFilePath);
+		} catch (Exception e) {
+			log.error("An error occurred while downloading file with pending requests", e);
+			responseEntity =  ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+		}
+		return responseEntity;
+	}
+
+
+	private Workbook getBlankWorkseet() {
+		Workbook pendingRequestWorkBook = new XSSFWorkbook();
+
+		Sheet pendingRequestSheet = pendingRequestWorkBook.createSheet("PendingUserRequest");
+		Row row = pendingRequestSheet.createRow(0);
+
+		Cell cell0 = row.createCell(0);
+		cell0.setCellValue("Full Name");
+
+		Cell cell1 = row.createCell(1);
+		cell1.setCellValue("Email");
+
+		Cell cell2 = row.createCell(2);
+		cell2.setCellValue("Mobile Number");
+
+		Cell cell3 = row.createCell(3);
+		cell3.setCellValue("Group");
+
+		Cell cell4 = row.createCell(4);
+		cell4.setCellValue("Designation");
+
+		Cell cell5 = row.createCell(5);
+		cell5.setCellValue("Gender");
+
+		Cell cell6 = row.createCell(6);
+		cell6.setCellValue("Category");
+
+		Cell cell7 = row.createCell(7);
+		cell7.setCellValue("Date of Birth (dd-mm-yyy)");
+
+		Cell cell8 = row.createCell(8);
+		cell8.setCellValue("Mother Tongue");
+
+		Cell cell9 = row.createCell(9);
+		cell9.setCellValue("Employee ID");
+
+		Cell cell10 = row.createCell(10);
+		cell10.setCellValue("Office Pin Code");
+
+		Cell cell11 = row.createCell(11);
+		cell11.setCellValue("External System ID");
+
+		Cell cell12 = row.createCell(12);
+		cell12.setCellValue("External System Name");
+
+		Cell cell13 = row.createCell(13);
+		cell13.setCellValue("Tags");
```

Comment:
this is wrong... 
this is for pending approval -- we will get approvals only for group and designation change... no other fields

Reply:
These column labels are now moved to csv file --as discussed.